stories/assets/js/FilterAndSearchBar.js

```diff
@@ -0,0 +1,54 @@
+const toggleFilter = function () {
+  $(document).on('click', '#toggle-search', () => {
+    $('#select-wrapper').toggleClass('show-filter');
+  });
+  const searchOption = $('.options li input');
+
+  const chipsWrapper = $('#search-filter #selected-chips');
+  $('.select-control').on('click', function () {
+    $(this).parent('.regions-select').toggleClass('open');
```

Comment:
@Ashish-srij Don't use region-select class as we are going remove this so please use other class or add some other generic class and use that